demos/package.xml

```diff
@@ -22,6 +22,9 @@
   <exec_depend>rmf_gazebo_plugins</exec_depend>
   <exec_depend>rmf_demo_tasks</exec_depend>
 
+  <exec_depend>joy</exec_depend>
+  <exec_depend>teleop_twist_joy</exec_depend>
```

Comment:
we will probably need to add these dependencies to the setup instructions and build testing scripts

Reply:
I was thinking we should enforce the use of `rosdep` in the build instructions. Most dependencies should be resolvable that way. I'll update the docs.